crt_portal/cts_forms/templatetags/get_shutdown_mode.py

```diff
@@ -0,0 +1,11 @@
+from django import template
+from utils.shutdown_mode import is_shutdown_mode
+
+register = template.Library()
+
+
+@register.simple_tag
+def get_shutdown_mode():
+    mode = is_shutdown_mode()
+    print("ShutdownMode = ", mode)
+    return mode
```

Comment:
```suggestion
    return is_shutdown_mode()
```

Reply:
Oops. Left this in for debugging. Good catch!